deployment/k8s/charts/aggregates/migrationAssistant/values.yaml

```diff
@@ -19,24 +20,37 @@ capture-proxy:
       yamlPath: "allow_insecure"
 
 migration-console:
+  snapshotVolumeEnabled: true
+  snapshotVolumePvc: "snapshot-volume-pvc"
   parameters:
     nothingImportant:
       value: false
```

Comment:
darn, just FYI - these should be deleted

Reply:
These got deleted with your latest PR 👍